aqi_monitor/utils/work_with_db.py

```diff
@@ -0,0 +1,84 @@
+from functools import reduce
+
```

Comment:
First of all, never ever assign names like "module to work with something" to modules and any kind of objects.  
I expected something like _db_connector_ or something similar.

Reply:
Fixed